index.js

```diff
@@ -33,7 +34,8 @@ function handleArgs(cmd, args, opts) {
 		stripEof: true,
 		preferLocal: true,
 		encoding: 'utf8',
-		reject: true
+		reject: true,
+		cleanup: false
```

Comment:
I was just conscious of this being a breaking change and thought you might not want that, happy to default it to `true`.